### ISSUE TITLE
Add-on doesn't show up at all despite numerous refresh, restarts etc.

### ISSUE THREAD

**Issue by reddcang** [trgkanki/remaining_time#36]
# Bug report

## Describe the bug: 

> A clear and concise description of what the bug is.: The add-on doesn't show up on any of my decks, tried configuring it by changing settings to 'true' but it has not shown any results so I restored it to default. (Please don't judge that I changed it, I am not a coder, I just want  to try and make this add-on work.) 

It doesn't work on my Anki. :(

## Reproduction step

> Steps to reproduce the behavior:

1. Open Anki
2. Press the deck you intend to study.
3. Press "Study Now"
4. No progress bar what so ever.

## Expected behavior

> I expected that there would be an estimated time at the top and a progress bar but there is known in BOTH day and night mode.

## Actual behavior

> Nothing appeared. (it is as if i did not install the add-on at all.)

## Anki/OS versions

- OS: macOS Sierra
- Anki version Version 2.1.33 (3f403040)
Python 3.8.1 Qt 5.9.7 PyQt 5.13.2

## Additional context

> maybe it's because my OS is too old. But I hope it doesn't hinder it from working or that there may be a solution to it. I am not familiar as how filing an issue works. This is my first time, but i hope I have provided enough info for it to be counted as an issue. Thank you.

Kind the screenshot here: (No progress bar or elapsed time etc)
![Screen Shot 2021-01-09 at 8 21 18 PM](https://user-images.githubusercontent.com/77195697/104091416-44ffb180-52b8-11eb-961c-b53886c511b2.png)

**Comment by reddcang** [comment]
English is not my main language. I apologize, this should be "I expected that there would be an estimated time at the top and a progress bar but there is 'none' in BOTH day and night mode."

**Comment by thiswillbeyourgithub** [comment]
Not the dev but did you try removing the addon, restarting anki, then reinstalling the addon? It sometimes solves weird issues.

**Comment by reddcang** [comment]
Hey, I immediately tried your suggestion. But unfortunately, it still
doesn't work.

Each time I reinstall it, after restarting Anki, it shows info about the
add-on: https://ankiweb.net/shared/info/1508357010 <- The same info there.

Hope this additional information helps. :) And thank you for taking the
time to reply and help

**Comment by phu54321** [comment]
Here I am. We had an exam last Friday, and we'll have a new exam within a month, so right now I'm quite busy preparing for the new exam. I hope I could be in touch within a week. No promise though, but I hope so. :)

**Comment by reddcang** [comment]
Hello,

It’s alright. Please take your time and i wish you well in your exams. :)

**Comment by EngerLar** [comment]
Hi,

I do have the exact same problem. The add-on does not work on my MacOS Sierra either (Anki-Version 2.1.35 alternative). It did work for some time but stopped to work in november or so last year. I already opened an issue here on github to adress my problem, but due to lack of time I wasn't able to answer the person who tried to help me, so the issue was closed (I'm sorry for that). Now revising the answer I got back then ("Not reproduced on macOS catalina, 2.1.35 alternative version. We need more debugging info.") I think the problem is that the add-on does not work on macOS Sierra. I also tried to download AnkiWebView Inspector, which also does not work on my Mac (when I click right, there is no "inspect"-option, only "copy").
So I think the MacOS **Sierra** version is the problem.

Thanks for your help!
